APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+<APE title here>
+================
```

Comment:
```suggestion
Guidelines for finding and equitably distributing Astropy Project funding
=========================================================================
```

Reply:
This APE is not about finding funding. It's about distributing funding. The process described is also not equitable. The process is based mostly on community input, not on equity.  The title I like is "Guidelines for Distributing Astropy Project Funding"

Reply:
How about "Distributing Astropy Project funding"?

Reply:
@hamogu's suggestion sounds good to me!

```suggestion
Distributing Astropy Project Funding
====================================
```